### PR TITLE
Add a manpage for atomic-openshift-installer

### PR DESCRIPTION
**Pull request by tbielawa**
Adds a man page.

### CODE REVIEW

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 1]
```diff
@@ -0,0 +1,167 @@
+atomic-openshift-installer(1)
+=============================
+:man source:   atomic-openshift-utils
+:man version:  %VERSION%
+:man manual:   atomic-openshift-installer
+
+
+NAME
+----
+atomic-openshift-installer - Interactive OpenShift Container Platform installer
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: Maybe set the initialism here to be used in the rest of the page:

"Interactive OpenShift Container Platform (OCP) installer"

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+SYNOPSIS
+--------
+atomic-openshift-installer [OPTIONS] COMMAND [OPTS]
+
+
+DESCRIPTION
+-----------
+
+**atomic-openshift-installer** makes the process for installing OSE or
+AEP easier by interactively gathering the data needed to run on each
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: s/OSE or AEP/OCP/

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+
+*-c*, *--configuration* 'PATH'::
+
+Provide an alternate 'PATH' to an 'installer.cfg.yml' file.
+
+
+*-a* 'DIRECTORY', *--ansible-playbook-directory* 'DIRECTORY'::
+
+Manually set the 'DIRECTORY' to look for Ansible playbooks in.
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: in which to look for Ansible playbooks.

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+
+*--ansible-log-path* 'PATH'::
+
+Specify the 'PATH' of the directory to save Ansible logs in.
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: in which to save Ansible logs.

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+The options specific to each command are described in the following
+sections.
+
+
+
+INSTALL
+-------
+
+The **install** command will guide you throgh steps required to
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: s/throgh/through/

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+install an OpenShift Container Platform. After all of the required
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: s/OpenShift Container Platform/OCP cluster/

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+information has been collected (target hosts, storage options,
+high-availability) the installation will begin.
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: high-availability),

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+*--gen-inventory*::
+
+Generate an Ansible inventory file and exit. The default location for
+the inventory file is '~/.config/openshift/hosts'.
+
+
+UNINSTALL
+---------
+
+The **uninstall** command will uninstall OpenShift Container Platform
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: OCP

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+from your target hosts. This command has no additional options.
+
+
+UPGRADE
+-------
+
+The **upgrade** command will upgrade a cluster of hosts to a newer
+version of the OpenShift Container Platform.
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: s/the OpenShift Container Platform/OCP/

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+
+*-l*, *--latest-minor*::
+
+Upgrade to the latest minor version. For example, if you are running
+version **3.2.1** then this could upgrade you to **3.2.2**.
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: FWIW, we call this a latest asynchronous release in the docs, with "minor version" meaning the Y-release (3.2, 3.3). But I see the option is `--latest-minor`, soooo.

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in [hunk 11]
```diff
@@ -0,0 +1,167 @@
...
+
+*-n*, *--next-major*::
+
+Upgrade to the latest major version. For example, if you are running
+version **3.2** then this could upgrade you to **3.3**.
```
Comment (adellape) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: Wouldn't major be 3 to 4? E.g., see https://access.redhat.com/support/policy/updates/openshift and (old article, but...) https://access.redhat.com/solutions/401413.

Comment (tbielawa) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: @adellape yes, when I queried @smunilla about those options earlier I had the same reaction. Shouldn't `major` refer to the **3** in `3.x.y`?

Comment (detiber) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: It's complicated? If we can find better verbiage, I'm all for it.

That said, the process from going from 3.n to 3.n+1 is far from minor. In the past for OpenShift we've informally used "major release" to mean a release that crosses channel boundaries and a "minor release" to mean a "z-stream" release. It definitely causes confusion when thinking about the traditional software versioning terminology, though.

File: utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in (hunk 10 above)
Comment (detiber) [inline utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in]: I'm not sure asynchronous release is correct here, either. An async errata wouldn't rev the the 'z' value of OpenShift, only a 'z-stream' errata would.

I'm not sure we want to expose that type of jargon into the docs, though.

**Comment by tbielawa** [comment]
I've got the spec file updated now to install the man page and Alex's comments have been addressed.

I don't think now is a good time to change the upgrade option flags.

@sdodson @detiber good to merge?

**Pull request merged**